src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -44,18 +46,32 @@ class SymfonyStyle extends OutputStyle
     private ProgressBar $progressBar;
     private int $lineLength;
     private TrimmedBufferOutput $bufferedOutput;
+    private OutputWrapperInterface $outputWrapper;
 
-    public function __construct(InputInterface $input, OutputInterface $output)
+    public function __construct(InputInterface $input, OutputInterface $output, OutputWrapperInterface $outputWrapper = null)
```

Comment:
Let's remove this argument.

Reply:
If the argument and the setters are removed, the property `OutputWrapper::$allowCutUrls` cannot be updated and becomes useless.